pkg/resource/grafana-agent-config/grafana-agent-config.go

```diff
@@ -81,6 +81,9 @@ loki.write "default" {
 		username = nonsensitive(remote.kubernetes.secret.credentials.data["logging-username"])
 		password = remote.kubernetes.secret.credentials.data["logging-password"]
 	}
+	tls_config {
+		insecure_skip_verify = ` + fmt.Sprintf("%v", lc.IsInsecureCA()) + `
```

Comment:
I assume we expect `true` or `false` here, using `%t` is preferable then.
```suggestion
		insecure_skip_verify = ` + fmt.Sprintf("%t", lc.IsInsecureCA()) + `
```

Reply:
Thanks :)